music21/instrument.py

```diff
@@ -63,10 +63,11 @@ def unbundleInstruments(streamIn, *, inPlace=False):
     if inPlace is True:
         s = streamIn
     else:
-        s = copy.deepcopy(streamIn)
+        s = streamIn.coreCopyAsDerivation('unbundleInstruments')
 
     for thisObj in s:
-        if 'Unpitched' in thisObj.classes:
+        if 'NotRest' in thisObj.classes:
+            # TODO(QUESTION) -- also unbundle each note of chord?
```

Comment:
yes, but that will be hard because we will need Voice objects to move each instrument.